custom/icds_reports/utils/aggregation_helpers/growth_monitoring_forms.py

```diff
@@ -24,6 +24,7 @@ def data_from_ucr_query(self):
         return """
             SELECT
                 DISTINCT child_health_case_id AS case_id,
+                supervisor_id AS supervisor_id,
```

Comment:
Need to be something like `LAST_VALUE(supervisor_id) OVER weight_child AS supervisor_id,`

Reply:
Updated